generators/client-2/templates/_package.json

```diff
@@ -4,31 +4,54 @@
   "description": "Description for <%= baseName %>",
   "private": true,
   "cacheDirectories": [
-    "node_modules",
-    "<%= MAIN_SRC_DIR %>bower_components"
+    "node_modules"
   ],
   "dependencies": {
-    "@angular/common": "2.0.1",
-    "@angular/compiler": "2.0.1",
-    "@angular/core": "2.0.1",
-    "@angular/forms": "2.0.1",
-    "@angular/http": "2.0.1",
-    "@angular/platform-browser": "2.0.1",
-    "@angular/platform-browser-dynamic": "2.0.1",
-    "@angular/upgrade": "2.0.1",
+    "@angular/common": "2.1.1",
+    "@angular/compiler": "2.1.1",
+    "@angular/core": "2.1.1",
+    "@angular/forms": "2.1.1",
+    "@angular/http": "2.1.1",
+    "@angular/platform-browser": "2.1.1",
+    "@angular/platform-browser-dynamic": "2.1.1",
+    "@angular/upgrade": "2.1.1",
     "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.10",
-    "core-js": "2.4.1",
-    "jquery": "3.1.0",
     "angular2-cookie": "1.2.5",
+    "angular2-template-loader": "0.6.0",
+    "angular-aria": "1.5.8",
+    "angular-bootstrap": "0.12.2",
+    "angular-cache-buster": "0.4.3",
+    "angular-cookies": "1.5.8",
+    "angular-dynamic-locale": "0.1.32",
+    "angular-loading-bar": "0.9.0",
+    "angular-resource": "1.5.8",
+    "awesome-typescript-loader": "2.2.4",
+    "bootstrap": "3.3.7",
+    "browser-sync-webpack-plugin": "1.1.3",
+    "core-js": "2.4.1",
+    "copy-webpack-plugin": "4.0.1",
+    "css-loader": "0.26.0",
+    "exports-loader": "0.6.3",
+    "file-loader": "0.9.0",
+    "html-webpack-plugin": "2.24.1",
+    "jquery": "2.2.2",
```

Comment:
@flaviencathala why is jquery downgraded to 2.x did you have any issue with 3.x?